### PR TITLE
Release 5.5.0

### PR DESCRIPTION
**Pull request by Cyberglamdring**

### CODE REVIEW

**Comment by HardNorth** [comment]
@Cyberglamdring Metrics-gathering?

**Comment by Cyberglamdring** [comment]
Metrics Gatherer tested on Alpha

File: reportportal/templates/metrics-gatherer-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        - name: POSTGRES_HOST
+          value: "{{ .Values.postgresql.endpoint.address }}"
+        - name: POSTGRES_PORT
+          value: "{{ .Values.postgresql.endpoint.port }}"
+        - name: POSTGRES_DB
+          value: "{{ .Values.postgresql.endpoint.dbName }}"
+        - name: POSTGRES_USER
+          value: "{{ .Values.postgresql.endpoint.user }}"
+        - name: POSTGRES_PASSWORD
+         {{ if .Values.postgresql.SecretName }}
```
Comment (HardNorth) [inline reportportal/templates/metrics-gatherer-deployment.yaml:35]: Please use `{{- ` just to not add empty lines in the result file.

File: reportportal/templates/metrics-gatherer-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            cpu: {{ .Values.metricsgatherer.resources.limits.cpu }}
+            memory: {{ .Values.metricsgatherer.resources.limits.memory }}
+{{ include "nodeSelector" . | indent 8 }}
+      securityContext:
+{{ toYaml .Values.metricsgatherer.securityContext | indent 8}}
+      serviceAccountName: {{ .Values.metricsgatherer.serviceAccountName }}
+{{- with .Values.tolerations }}
+      tolerations: 
+{{- toYaml . | nindent 8 }}
+{{- end }}
```
Comment (HardNorth) [inline reportportal/templates/metrics-gatherer-deployment.yaml:70]: Newline

File: reportportal/templates/metrics-gatherer-service.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    infoEndpoint: {{ $.Values.rp.infoEndpoint | default "/info"}}
+spec:
+  ports:
+  - name: headless
+    port: 8585
+    protocol: TCP
+    targetPort: 8585
+  selector:
+    component: {{ include "reportportal.fullname" . }}-metrics-gatherer
+  type: ClusterIP
```
Comment (HardNorth) [inline reportportal/templates/metrics-gatherer-service.yaml:17]: Newline

**Review by HardNorth — CHANGES_REQUESTED** [review]

File: reportportal/templates/metrics-gatherer-deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+          value: "{{ .Values.postgresql.endpoint.dbName }}"
+        - name: POSTGRES_USER
+          value: "{{ .Values.postgresql.endpoint.user }}"
+        - name: POSTGRES_PASSWORD
+         {{- if .Values.postgresql.SecretName }}
+          valueFrom:
+            secretKeyRef:
+              name: "{{ .Values.postgresql.SecretName }}"
+              key: "postgresql-password"
+        {{ else }}
```
Comment (HardNorth) [inline reportportal/templates/metrics-gatherer-deployment.yaml:40]: Please put `{{- ` here too.

**Review by HardNorth — CHANGES_REQUESTED** [review]

**Review by HardNorth — APPROVED** [review]

**Review by pbortnik — APPROVED** [review]

**Pull request merged**